internal/services/instances.go

```diff
@@ -178,6 +210,195 @@ func (s *ServiceClient) ListInNamespace(ctx context.Context, namespace string) (
 func (s *ServiceClient) list(ctx context.Context, namespace string) (models.ServiceList, error) {
 	serviceList := models.ServiceList{}
 
+	listOpts := metav1.ListOptions{}
+	if namespace == "" {
+		listOpts.LabelSelector = fmt.Sprintf("%s,%s", ServiceNameLabelKey, CatalogServiceLabelKey)
+	} else {
+		listOpts.LabelSelector = fmt.Sprintf(
+			"%s,%s",
+			ServiceNameLabelKey,
+			CatalogServiceLabelKey,
+		)
+	}
```

Comment:
Are these the same?

Reply:
Maybe? Will check ...

Reply:
The look indeed the same. Will check if my rewriting the code introduced an error here.

Reply:
The original code is https://github.com/epinio/epinio/blob/15b90264ffb9aa02948bbdb253fe9e54d247fc77/internal/services/instances.go#L182-L191 ... And it was different for namespace vs all, because the helm chart specs were all stored in the epinio namespace, with a label telling us which user namespace it was for (the target namespace).

For the new form the secrets for a service are stored in the namespace they belong to. The change is good, and can be simplified, as you saw. Will do that.

Reply:
Fixed. Commit 773b38c89c0f0c4d0b82f68d6daa41cb20de1427